src/sentry/utils/outcomes.py

```diff
@@ -135,26 +64,6 @@ def track_outcome(
 
     timestamp = timestamp or to_datetime(time.time())
 
```

Comment:
Please update the docstring of the method that still says we are updating redis.